<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_workout_view</class>
 <widget class="QDialog" name="add_workout_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Workouts - Workout Tracker</string>
  </property>
  <widget class="QPushButton" name="back_to_main_view">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_status_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Add Workout&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="workout_name_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Workout Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_workout_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="workout_day_combo_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Workout Day</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>workout_name_text</tabstop>
  <tabstop>add_workout_button</tabstop>
  <tabstop>back_to_main_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
